cirq-web/README.rst

```diff
@@ -0,0 +1,48 @@
+
+
+`Cirq <https://quantumai.google/cirq>`__ is a Python library for writing, manipulating, and optimizing quantum
+circuits and running them against quantum computers and simulators.
+
+This module is **cirq-web**, which allows users to take advantage of Typescript-based 3d visualization tools
+and features in Cirq. cirq-web also provides a development environment for contributors to create and add 
+their own visualizations to the module.
+
+Documentation
+-------------
+All current documentation for cirq-web can be found in the README files located in this module's subdirectories.
+
+Below is a quick example of how to generate a portable 3d rendering of a bloch sphere using cirq-web:
+
+.. code-block:: python
+
+    from cirq_web import BlochSphere
+    sphere = BlochSphere()
+    sphere.generate_HTML_file()
+
+This will create the file in the current working directory. There are additional options to specify the
+output directory, open the visualization in a browser, etc. 
+
+You can also view and interact with a bloch sphere in a Colab or Jupyter notebook setting
+with the following:
+
+.. code-block:: python
+
+    from cirq_web import BlochSphere
+    sphere = BlochSphere()
+    display(sphere)
+
+Note that you can pass a state vector into the :code:`BlochSphere()` constructor to view a particular
```

Comment:
My thought process is that since this is the PyPI readme, and in production users won't be accessing the typescript at all, its safe to provide examples using just the Python side.